.drone.star

```diff
@@ -728,13 +740,13 @@ minio_mc_environment = {
 
 go_step_volumes = [{
     "name": "server",
-    "path": "/srv/app",
+    "path": "%s" % dir["app"],
 }, {
     "name": "gopath",
     "path": "/go",
 }, {
     "name": "configs",
-    "path": "/srv/config",
+    "path": dir["config"],
```

Comment:
Any preferences in using two different ways?
Others look good!

Reply:
my mistake